### PR TITLE
Adding pwnsoni

### PR DESCRIPTION
**Pull request by pwnsoni**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/pwnsoni/pwnsoni.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Pawan Soni # No longer than 28 characters
+institution: JSS Academy of Technical Education, Noida 🚩 # no longer than 58 characters
+profile_pic: pic.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
+quote: "Never say never" # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.
```
Comment (Mohammed-Shoaib) [inline _data/pwnsoni/pwnsoni.md:5]: Remove double quotes from begin and end.

Comment (pwnsoni) [reply _data/pwnsoni/pwnsoni.md]: Updated the required changes.

**Comment by jeffrafter** [comment]
@pwnsoni I'm curious... how did you make your PNG? The file size is incredibly small and all of my attempts to resize it just make it bigger (usually 252KB)! Thanks for any insight 🙌

**Comment by Mohammed-Shoaib** [comment]
@jeffrafter Use an [image compressor](https://www.iloveimg.com/compress-image)?
Even so, 252KB really isn't much, it's a small file size already. :ok_man:

**Pull request merged**